oioioi/mp/controllers.py

```diff
@@ -97,7 +97,7 @@ def can_change_terms_accepted_phrase(self, request):
 
 
 class MPContestController(ProgrammingContestController):
-    description = _("Mistrz Programowania")
+    description = _("Master of Programming")
```

Comment:
Not sure this name should be translated to English. It is more PR related question rather than an issue.